### DISCUSSION TITLE
一个域名对应多个IP如何使用，教程没看明白

### DISCUSSION THREAD

**Discussion by a13312127532** [jeessy2/ddns-go#1221] [Q&A]
一个域名对应多个IP如何使用，教程没看明白，我用的是cloudflare，那个自定义参数该填在哪，我没看懂，感谢大佬！

**Comment by paulownia99** [comment]
同问：@放哪里，re表达式要匹配一个地址多次还是同时匹配多个地址？用DDNS-go通过网卡获取匹配第一个(主机)和第二个地址（路由器）分别给阿里两个子域名a.b.cn和b.b.cn怎么设置正则^2409(:[0-9a-fA-F]{1,4}){3}:*[0-9a-fA-F]{1,4}$和Domains，
a.b.cn?RecordId=0
b.b.cn?RecordId=1
这样对吗？@不知道加到哪里，会报错
